src/javaRosa.js

```diff
@@ -824,16 +824,17 @@ define([
         options.id = id;
 
         var widget = widgets.base(mug, options);
-        var $input = $("<textarea></textarea>")
+        var $input = $("<div contenteditable>")
             .attr("name", widget.id)
-            .attr("rows", "2")
-            .addClass('input-block-level itext-widget-input')
+            .addClass('fake-textarea input-block-level itext-widget-input')
```

Comment:
Need to figure out a good way to specify # of rows in a div. May just need to go with height though.